core/src/main/java/io/lionweb/lioncore/java/serialization/AbstractSerialization.java

```diff
@@ -22,7 +23,6 @@
  * and the actual physical formats is done in other classes.
  */
 public abstract class AbstractSerialization {
-  public static final String DEFAULT_SERIALIZATION_FORMAT = "2023.1";
```

Comment:
Do we care about compatibility with existing implementations? This breaks existing users.

Reply:
I have restored it and marked as deprecated